syntax = "proto3";

option go_package = "server/djentry";

message UserInfoRequest {
	string user_id = 1;
}

message UserInfoResponse {
	string msg = 1;
}

service DjentryStatistics {
	rpc GetUserInfo(UserInfoRequest) returns (UserInfoResponse);
}
